src/script/view_model/CallingViewModel.ts

```diff
@@ -478,7 +478,7 @@ export class CallingViewModel {
   }
 
   private showRestrictedConferenceCallingModal() {
-    if (this.selfUser().inTeam()) {
+    if (this.teamState.isInTeam(this.selfUser())) {
```

Comment:
Here the call to `this.selfUser()` was typed as `any` because of the `ko.Subscribable`. 
Converting to a `ko.Observable` fixed the type issue